### PR TITLE
ncm-metaconfig: moab: use stricter schema

### PR DESCRIPTION
**Pull request by stdweird**
Switch to a more well defined schema (instead of a `string{}`)

### CODE REVIEW

**Comment by jrha** [comment]
"refine attributes" doesn't give me enough to understand the context of this change.

**Comment by stdweird** [comment]
@jrha is that enough?

**Comment by jrha** [comment]
Yep.

File: ncm-metaconfig/src/main/metaconfig/moab/pan/schema.pan [hunk 1]
```diff
@@ -3,9 +3,41 @@ declaration template metaconfig/moab/schema;
 include 'pan/types';
 
 @documentation{
-    The legacy moab service corrsponds to the old moab component schema, with only difference
-        include is a list of absolute filenames, and does not support contents or ok
+    The legacy moab service corresponds to the old moab component schema, with only difference
+    include is a list of absolute filenames, and does not support contents or ok
```
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/moab/pan/schema.pan]: This sentence doesn't make any sense.

**Comment by stdweird** [comment]
@jrha rephrased the annotation documentation

**Pull request merged**